src/js/view/controls/rightclick.js

```diff
@@ -16,8 +16,9 @@ function createDomElement(html) {
 }
 
 export default class RightClick {
-    constructor(infoOverlay) {
+    constructor(infoOverlay, api) {
```

Comment:
Pass in a callback on `setup`. Let's keep the constructor clean and not add properties we don't need to the instance.